src/renderer/lib/state.js

```diff
@@ -205,7 +210,13 @@ function load (cb) {
     if (err) return cb(err)
     const state = getDefaultState()
     state.saved = saved
-    migrations.run(state)
+
+    if (process.type === 'renderer') {
+      // Perf optimization: Save require() calls in the main process
```

Comment:
Ah,  I didn't see this @dcposch. Maybe after this gets merged I'll re-look #997.